portal/templates/gil/symptom-tracker.html

```diff
@@ -1,41 +1,41 @@
-          <p class="module__body">Every time you complete the tracking questionnaire it adds data to your graph, shows you how your experience compares with other men, and provides relevant tips.</p><img src="{{url_for('static', filename='img/Symptom_Tracker_Graph.png') }}" alt="Symptom Tracker Graph">
+          <div class="accent-heading">{% trans %}Objective N<em>o</em>6: Custom Tools – Symptom Tracker {% endtrans %}</div>
+          <h3 class="module__head">{{_("Monitoring and Tracking")}}</h3>
+          <p class="module__body">{{_("We’ve created a tool that asks you questions about your symptoms and side-effects throughout your prostate cancer journey from diagnosis through recovery.")}}</p>
+          <p class="module__body">{{_("Every time you complete the tracking questionnaire it adds data to your graph, shows you how your experience compares with other men, and provides relevant tips.")}}</p><img src="{{url_for('static', filename='img/Symptom_Tracker_Graph.png') }}" alt="{{_('Symptom Tracker Graph')}}">
           <div class="next-down">
-            <p class="next-down__copy">We’ve created a tool to track your prostate cancer treatment side effects over time and provide personal guidance.</p>
+            <p class="next-down__copy">{{_("We’ve created a tool to track your prostate cancer treatment side effects over time and provide personal guidance.")}}</p>
           </div>
           <div class="icon-box-wrap">
             <div class="icon-box-wrap__main icon-box icon-box--theme-compass icon-box--theme-brown">
-              <div class="icon-box-wrap__main__left"><strong class="icon-box__subhead">Tool N<em>o</em>2: Monitoring</strong>
-                <h4 class="icon-box__head">Symptom Tracker</h4><span class="icon-box__lowhead">Questionnaire / 10 Mins</span>
+              <div class="icon-box-wrap__main__left"><strong class="icon-box__subhead">{% trans %}Tool N<em>o</em>2: Monitoring {% endtrans %}</strong>
+                <h4 class="icon-box__head">{{_("Symptom Tracker")}}</h4><span class="icon-box__lowhead">{{_("Questionnaire / 10 Mins")}}</span>
                 <svg class="icon-box__icon" width="90" height="90">
                   <use xlink:href="#ST-Graph"></use>
                 </svg>
               </div>
-              <div class="icon-box-wrap__main__right"><a class="icon-box__button icon-box__button--main symptom-tracker-link {% if not user %}icon-box__button--disabled{% endif %}" href="{{PORTAL}}{%if user%}/home{%else%}/{%endif%}"><span>Start<figure></figure></span></a></div>
+              <div class="icon-box-wrap__main__right"><a class="icon-box__button icon-box__button--main symptom-tracker-link {% if not user %}icon-box__button--disabled{% endif %}" href="{{PORTAL}}{%if user%}/home{%else%}/{%endif%}"><span>{{_("Start")}}<figure></figure></span></a></div>
```

Comment:
No `PORTAL` please :)
That conditional home page link is probably not necessary- I think "/" should redirect to "/home"